app/main/suppliers.py

```diff
@@ -63,7 +63,7 @@ def suppliers_list_by_prefix():
     page = process_page(request.args.get('page', default=u"1"))
 
     try:
-        api_result = data_api_client.find_suppliers(api_prefix, page, 'gcloud')
+        api_result = data_api_client.find_suppliers(api_prefix, page, 'g-cloud')
```

Comment:
I think this should go in as is. It's fixing a specific bug and we're not absolutely sure we know how the DOS pages are going to work yet.